src/Components/Aspire.Milvus.Client/README.md

```diff
@@ -0,0 +1,105 @@
+# Aspire.Milvus.Client library
+
+Registers a [MilvusClient](https://github.com/milvus-io/milvus-sdk-csharp) in the DI container for connecting to a Milvus server.
+
+## Getting started
+
+### Prerequisites
+
+- Milvus server and connection string for accessing the server API endpoint.
+
+### Install the package
+
+Install the .NET Aspire Milvus Client library with [NuGet](https://www.nuget.org):
+
+```dotnetcli
+dotnet add package Aspire.Milvus.Client
+```
+
+## Usage example
+
+In the _Program.cs_ file of your project, call the `AddMilvusClient` extension method to register a `MilvusClient` for use via the dependency injection container. The method takes a connection name parameter.
+
+```csharp
+builder.AddMilvusClient("milvus");
+```
+
+## Configuration
+
+The .NET Aspire Milvus Client component provides multiple options to configure the server connection based on the requirements and conventions of your project.
+
+### Use a connection string
+
+When using a connection string from the `ConnectionStrings` configuration section, you can provide the name of the connection string when calling `builder.AddMilvusClient()`:
+
+```csharp
+builder.AddMilvusClient("milvus");
+```
+
+And then the connection string will be retrieved from the `ConnectionStrings` configuration section:
+
+```json
+{
+  "ConnectionStrings": {
+    "milvus": "Endpoint=http://localhost:19530;Key=root:123456!@#$%"
```

Comment:
Why does the `Key` have `root:` in it? But it doesn't below in the json example. And then it does again in the C# example.

Reply:
In Milvus terms the "apiKey" is a user:pass name (root:Milvus being default)...rather than a traditional apiKey aspect (which may exist for some hosted/managed instances of Milvus).  Can make this consistent for sure, but also trying not to have our readme document the milvus default -- it _can_ be configured after the fact so someone could have a separate u/p if they have a script that does the update.